gordo_components/builder/build_model.py

```diff
@@ -248,12 +249,12 @@ def _build(self) -> Tuple[Optional[sklearn.base.BaseEstimator], dict]:
                 scaler = self.evaluation_config.get("scoring_scaler")
                 metrics_dict = self.build_metrics_dict(metrics_list, y, scaler=scaler)
 
+                # Generate metadata about CV train, test splits
+                split_obj = TimeSeriesSplit(n_splits=3)
```

Comment:
Do we have a way of specifying the number of splits in the evaluation config? Or should we make that into an issue if not? I do not like the hardcoded way of setting the number of splits.

Reply:
I agree that should be changed, but I didn't think that should be for this PR.

Reply:
Yeah, I also see it as a separate issue, just shining some light at it.